cardano-c/examples/README.md

```diff
@@ -0,0 +1,45 @@
+# Creating a wallet
+
+This example shows how to create a wallet from english mnemonics
+
+```C
+const char *alias = "Awesome Wallet";
```

Comment:
```suggestion
const char *alias = "Awesome Account";
```

It is the alias of an account, not a wallet. Just to make sure the readers are not confused but the wording